flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/calls/ScalarOperators.scala

```diff
@@ -911,6 +911,64 @@ object ScalarOperators {
     }
   }
 
+  def generateRand(
```

Comment:
The ScalarOperators class is intended for basic operations such as plus, minus, equals etc. Can you add the functions to `FunctionGenerator` class?